CONTRIBUTING.md

```diff
@@ -3,4 +3,4 @@
 In favor of active development we accept contributions from everyone. You can contribute by submitting a bug, creating pull requests or even by improving documentation.
 
 Below is the guide to be followed strictly before submitting your pull requests.
-http://adonisjs.com/docs/2.0/contributing
+http://adonisjs.com/docs/3.0/community
```

Comment:
Can u please remove the `version number`. http://adonisjs.com/docs/community intead of this ~http://adonisjs.com/docs/3.0/community~. I have an internal redirect to the most latest version if version is not defined